example/index.js

```diff
@@ -2,7 +2,6 @@
  * @format
  */
 
-import 'react-native-get-random-values';
```

Comment:
Don't we want to promote adding this? Or is it no longer needed?

Reply:
Is this no longer needed?  I definitely don't want an example to start with warnings about this.

Reply:
> Is this no longer needed?

I think it is, that's why I'm curious why you suggest removing it? 😄

Reply:
Sorry, I read this as your suggestion :D I did not intend to remove it.  Unsure how that happened.  Good find!